erpnext/patches/v8_0/create_address_doc_from_address_field_in_company.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	# new field address_html is created in place of address field for the company's address
+	# patch for moving the address details in the address doc
+	company_list = frappe.db.sql('''select name, address from `tabCompany` where address is not null''', as_dict=1)
```

Comment:
@manassolanki,

Please check if the address column is available or not then use `frappe.db.sql`